package.json

```diff
@@ -29,7 +29,9 @@
       "hydrogen:run-cell-and-move-down"
     ]
   },
-  "scripts": {},
+  "scripts": {
+    "install": "npm rebuild --build-from-source"
```

Comment:
`rm -rf node_modules` and `apm install` works 👍 

I override the install option only because of the `--build-from-source` flag.